Examples/SessionTools-iOSExample/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0.2</string>
+	<string>1.0.3</string>
```

Comment:
Let's keep the version set to `1.0.2` since we'll do a separate PR for the actual release.

Reply:
Sounds good!